README.adoc

```diff
@@ -585,32 +593,30 @@ The default properties that are defined in the [hotspot file=0]`pom.xml` file ar
 | *Property*                      | *Description*
 | [hotspot=sysKubeService file=0]`system.ip` | Name of the {kube} Service wrapping the `system` pods, `system-service` by default.
 | [hotspot=invKubeService file=0]`inventory.ip` | Name of the {kube} Service wrapping the `inventory` pods, `inventory-service` by default.
-| [hotspot=sysHttpPort file=0]`sys.service.http.port`    | The HTTP port for the {kube} Service `system-service`, 9080 by default.
-| [hotspot=invHttpPort file=0]`inv.service.http.port`    | The HTTP port of the {kube} Service `inventory-service`, 9081 by default.
+| [hotspot=sysHttpPort file=0]`sys.http.port`    | The HTTP port for the {kube} Service `system-service`, 9080 by default.
+| [hotspot=invHttpPort file=0]`inv.http.port`    | The HTTP port of the {kube} Service `inventory-service`, 9081 by default.
 |===
 
 
 === Running the tests
 
-Run the Maven `verify` goal to test your microservices by replacing the `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]`
+Run the Maven `failsafe:integration-test` goal to test your microservices by replacing the `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]`
```

Comment:
Update to 
"Run the `failsafe:integration-test` Maven goal to test your microservices by replacing `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]` with the values that were determined in the previous section."

Reply:
I think saying Maven first makes more sense.